system-tests/fixtures/api/complex-api/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "author": "f0b00n7",
+  "author": "blemair",
```

Comment:
It's kinda annoying that this file gets touched and this field updated based on the local username of whoever runs the script. The author/committer ID is already tracked in source control, there should be no need to have this _inside_ of `package.json`.

Suggestion:
Update the two functions that call `ern create-api` in `regen-fixtures.js` to also include a `-u "generated"` flag.

Reply:
Agreed, that's a good idea :+1:

Reply:
#1418